driver/src/main/scala/org/mongodb/scala/MongoCompressor.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.scala
+
+import com.mongodb.{MongoCompressor => JMongoCompressor}
+
+/**
+ * Metadata describing a compressor to use for sending and receiving messages to a MongoDB server.
+ *
+ * @since 3.6
+ * @note Requires MongoDB 3.4 or greater
+ */
+object MongoCompressor {
+
+  /**
+   * Create an instance for snappy compression.
+   *
+   * @return A compressor based on the snappy compression algorithm
+   */
+  def createSnappyCompressor: MongoCompressor = JMongoCompressor.createSnappyCompressor()
+
+  /**
+   * Create an instance for zlib compression.
```

Comment:
Perhaps note that this requires server 3.6 or later

Reply:
Done

Reply:
Done